.github/actions/bazel-test-all/action.yaml

```diff
@@ -83,28 +83,51 @@ runs:
                 RUN_ON_DIFF_ONLY="false"
             fi
 
-            # Some environment variables needed by diff.sh
-            export BAZEL_TARGETS='${{ inputs.BAZEL_TARGETS }}'
-            export BAZEL_COMMAND='${{ inputs.BAZEL_COMMAND }}'
-
-            if [[ "${RUN_ON_DIFF_ONLY:-}" == "true" ]]; then
-                # get bazel targets that changed within the MR
-                BAZEL_TARGETS=$("${CI_PROJECT_DIR:-}"/ci/bazel-scripts/diff.sh)
-            fi
-
-            # if bazel targets is empty we don't need to run any tests
-            if [ -z "${BAZEL_TARGETS:-}" ]; then
-                echo "No bazel targets to build"
-                exit 0
-            fi
 
             bazel_args=( )
 
             if [[ $release_build == "true" ]]; then
               bazel_args+=( --config=release )
             fi
 
+            BAZEL_TARGETS='${{ inputs.BAZEL_TARGETS }}'
+
+            bazel_targets=( )
+            for tgt in $BAZEL_TARGETS; do
+              bazel_targets+=( "$tgt" )
+            done
+
+
+            if [[ "${RUN_ON_DIFF_ONLY:-}" == "true" ]]; then
+                target_pattern_file=$(mktemp) # TODO: remove (or upload?)
```

Comment:
I originally thought of removing the file but realized it might be interesting to upload it as an artifact

Reply:
Note that the full bazel invocation is also saved in BuildBuddy.

Reply:
would that give us the content of `--target_pattern_file`?